packaging/debian-unstable/patches/flags-compat.patch

```diff
@@ -0,0 +1,20 @@
+Description: update a test that fails with a new golang-go-flags-dev
+Author: Michael Hudson-Doyle <michael.hudson@ubuntu.com>
+Origin: vendor
+Forwarded: not-needed
+Last-Update: 2017-08-15
+---
+This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
+
+--- a/cmd/snap/cmd_interfaces_test.go
++++ b/cmd/snap/cmd_interfaces_test.go
+@@ -62,7 +62,8 @@
+       -i=                          Constrain listing to specific interfaces
+ 
+ [interfaces command arguments]
+-  <snap>:<slot or plug>:           Constrain listing to a specific snap or snap:name
++  <snap>:<slot or plug>:           Constrain listing to a specific snap or
++                                   snap:name
```

Comment:
Can we please fix the test instead, so that it works with both, or even remove the test for the time being.

Reply:
It's gone in master.